websites/self-learning-smoke-test/src/layouts/learning-home.js

```diff
@@ -60,8 +64,8 @@ const HeaderContainer = styled.div`
   /* Override some styles of the original <LayoutHeader> */
   header {
     grid:
-      [row1-start] 'header-top-menu header-searchbox' ${designSystem.dimensions
-        .heights.header} [row1-end]
+      [row1-start] '${GRID_ID_HEADER_TOP_MENU} ${GRID_ID_HEADER_SEARCHBOX}' ${designSystem
```

Comment:
Is this the actual fix? @ColinRosati

Reply:
Theres few files where I remove the hard coded text for the shared vars. This is just a quality of life refactor. Not the functional fix

Reply:
ah, yes, my bad. Thank you